### DISCUSSION TITLE
Use UUID on Send Notification operation not intuitive

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15797] [Feature Requests]
@directus/team I realize that the UUID option here enables us to more easily select takeFlow Object data. But if we don't want to programmatically select users from the Flow Object, this is somewhat unintuitive, and relatively hard info to get off of the User Directory. I really think we should also include the option to open the User Directory in a side drawer like we have in roles and permissions.

<img width="825" alt="Screen Shot 2022-09-28 at 3 49 24 PM" src="https://user-images.githubusercontent.com/62306630/192886041-5f423d44-5f2d-4dff-97e9-65846df8ab88.png">

<img width="1402" alt="Screen Shot 2022-09-28 at 3 58 37 PM" src="https://user-images.githubusercontent.com/62306630/192887449-6070b0a5-2b35-4953-a939-116919764d88.png">

**Comment by br41nslug** [comment]
Great suggestion, we recently did a similar change for selecting flows in the "trigger flow" operation.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
